tests/testthat/test-ggridges.R

```diff
@@ -0,0 +1,342 @@
+skip_if_not_installed("ggridges")
+library(ggridges)
+
+test_that(
+  desc = "ggridges basic ridgelines", 
+  code = {
+    
+    # simple ridge plot
+    data <- data.frame(x = 1:5, y = rep(1, 5), height = c(0, 1, 3, 4, 2))
+    p <- ggplot(data, aes(x, y, height = height)) + geom_ridgeline()
+    
+    expect_no_error(
+      p2 <- plotly_build(p)
+    )
+    p2 <- ggplotly(p)
```

Comment:
What's your reasoning for calling both `plotly_build()` and `ggplotly()` (and only testing no error for `plotly_build()`)?

Reply:
I don't have a very good idea of how to test in for plotly. I wanted to use at least the visual test, which is why I use `p2 <- ggplotly(p)` and then pass it to `expect_doppelganger`, but threw in the `expect_no_error` to at least ensure that it builds without error. Although I guess it would fail in the next line anyway. 
Should I just keep the `ggplotly` and remove the `plotly_build`?

Reply:
Yea, I think you can rid of all the `expect_no_error(plotly_build(p))`

Reply:
done